megatron/mpu/layers.py

```diff
@@ -547,8 +548,10 @@ def set_parallel_output(self, value: bool):
         )  # if gather_output is True, parallel output is False, so we set the opposite
 
     def forward(self, input_):
+
+        # Y_logits = W_unembed * X / m_width
         if self.use_mup and self.mup_rescale_parameters:
-            input_ /= self.width_mult()
+            input_ /= self.m_width
```

Comment:
This is unnecessary and should be removed. The only adjustments that need to be made are to the LR and init_std. This will apply an adjustment twice. I think its safe to remove all muP stuff from this file

Reply:
I intended to implement the formula `Y_logits = W_unembed * X / m_width` from this table, specifically `output_logits_multiplier`
<img width="941" alt="image" src="https://github.com/EleutherAI/gpt-neox/assets/5774558/027b2001-3e42-4ece-8dd3-eadb92f2dab1">

Reply:
Nvm, I realized where I was wrong. `Output logits multiplier` refers to the output embedding layer and not the output of the intermediate layers. I've removed the scaling in `layers.py`